Index: xbmc-20110405/xbmc/cores/DllLoader/exports/util/EmuFileWrapper.h
===================================================================
--- xbmc-20110405.orig/xbmc/cores/DllLoader/exports/util/EmuFileWrapper.h	2011-05-09 11:59:01.013603003 +0200
+++ xbmc-20110405/xbmc/cores/DllLoader/exports/util/EmuFileWrapper.h	2011-05-09 11:59:52.281603044 +0200
@@ -29,10 +29,14 @@
 
 class CMutex;
 
-#if defined(_LINUX) && !defined(__APPLE__) && !defined(__FreeBSD__)
+#if defined(_LINUX) && !defined(__APPLE__) && !defined(__FreeBSD__) && !defined(__UCLIBC__)
 #define _file _fileno
 #endif
 
+#if defined(__UCLIBC__)
+#define _file __filedes
+#endif
+
 #define MAX_EMULATED_FILES    50
 #define FILE_WRAPPER_OFFSET   0x00000100
 
