Index: feeds/packages/libs/libsdl/Makefile
===================================================================
--- feeds/packages/libs/libsdl/Makefile	(revision 26783)
+++ feeds/packages/libs/libsdl/Makefile	(working copy)
@@ -120,6 +120,18 @@
 	--disable-alsa
 endif
 
+ifneq ($(CONFIG_FEATURE_drawing-backend_DirectFB),)
+  CONFIGURE_ARGS+= \
+	--disable-video-x11 \
+	--without-x \
+	--enable-video-directfb
+else
+  CONFIGURE_ARGS+= \
+	--enable-video-x11 \
+	--with-x \
+	--disable-video-directfb
+endif
+
 CONFIGURE_VARS+= \
 		ac_cv_path_DIRECTFBCONFIG=no \
 

