Index: feeds/packages/Xorg/font/dejavu-fonts-ttf/Makefile
===================================================================
--- feeds/packages/Xorg/font/dejavu-fonts-ttf/Makefile	(revision 27590)
+++ feeds/packages/Xorg/font/dejavu-fonts-ttf/Makefile	(working copy)
@@ -56,7 +56,6 @@
   SUBMENU:=font
   CATEGORY:=Xorg
   URL:=http://dejavu.sourceforge.net/
-  DEPENDS:=@DISPLAY_SUPPORT
 endef
 
 $(foreach file,$(FILES),$(eval $(call PartGen,$(file))))

