Index: tvheadend-20111704/Makefile
===================================================================
--- tvheadend-20111704.orig/Makefile	2011-04-19 13:30:08.624103591 +0200
+++ tvheadend-20111704/Makefile	2011-04-19 13:30:21.576103410 +0200
@@ -24,7 +24,7 @@
 
 PROG=${BUILDDIR}/tvheadend
 
-CFLAGS  = -Wall -Werror -Wwrite-strings -Wno-deprecated-declarations 
+CFLAGS  ?= -Wall -Werror -Wwrite-strings -Wno-deprecated-declarations 
 CFLAGS += -Wmissing-prototypes
 LDFLAGS += -lrt -ldl
 
