Index: tvheadend-20111704/src/access.c
===================================================================
--- tvheadend-20111704.orig/src/access.c	2011-04-19 13:48:38.824103593 +0200
+++ tvheadend-20111704/src/access.c	2011-04-19 13:49:45.276103579 +0200
@@ -210,11 +210,11 @@
 
   if(superuser_username != NULL && superuser_password != NULL) {
 
-    SHA_Init(&shactx);
-    SHA_Update(&shactx, (const uint8_t *)superuser_password,
+    SHA1_Init(&shactx);
+    SHA1_Update(&shactx, (const uint8_t *)superuser_password,
 	       strlen(superuser_password));
-    SHA_Update(&shactx, challenge, 32);
-    SHA_Final(d, &shactx);
+    SHA1_Update(&shactx, challenge, 32);
+    SHA1_Final(d, &shactx);
 
     if(!strcmp(superuser_username, username) && !memcmp(d, digest, 20))
       return 0xffffffff;
@@ -229,8 +229,8 @@
     if((b & ae->ae_netmask) != ae->ae_network)
       continue; /* IP based access mismatches */
 
-    SHA_Init(&shactx);
-    SHA_Update(&shactx, (const uint8_t *)ae->ae_password,
+    SHA1_Init(&shactx);
+    SHA1_Update(&shactx, (const uint8_t *)ae->ae_password,
 	      strlen(ae->ae_password));
     SHA1_Update(&shactx, challenge, 32);
     SHA1_Final(d, &shactx);
Index: tvheadend-20111704/src/trap.c
===================================================================
--- tvheadend-20111704.orig/src/trap.c	2011-04-19 13:59:08.508103594 +0200
+++ tvheadend-20111704/src/trap.c	2011-04-19 14:00:12.712103603 +0200
@@ -246,9 +246,9 @@
       char *m = malloc(st.st_size);
       if(m != NULL) {
 	if(read(fd, m, st.st_size) == st.st_size) {
-	  SHA_Init(&binsum);
-	  SHA_Update(&binsum, (void *)m, st.st_size);
-	  SHA_Final(digest, &binsum);
+	  SHA1_Init(&binsum);
+	  SHA1_Update(&binsum, (void *)m, st.st_size);
+	  SHA1_Final(digest, &binsum);
 	}
 	free(m);
       }
