Index: sc-r61975953edd0/FFdecsa/Makefile
===================================================================
--- sc-r61975953edd0.orig/FFdecsa/Makefile	2011-04-11 12:34:28.112347998 +0200
+++ sc-r61975953edd0/FFdecsa/Makefile	2011-04-11 12:34:45.956348039 +0200
@@ -37,7 +37,7 @@
           parallel_128_sse2.h \
           parallel_128_sse.h
 
-all: FFdecsa.o FFdecsa_test.done
+all: FFdecsa.o FFdecsa_test
 
 %.o: %.c
 	$(COMPILER) $(FLAGS) -DPARALLEL_MODE=$(PARALLEL_MODE) -c $<
