Index: sc-r61975953edd0/Makefile
===================================================================
--- sc-r61975953edd0.orig/Makefile	2011-04-14 13:23:56.840193003 +0200
+++ sc-r61975953edd0/Makefile	2011-04-14 13:24:43.876192581 +0200
@@ -185,7 +185,7 @@
 	$(AR) r $@ $(OBJS)
 
 $(FFDECSA) $(FFDECSATEST): $(FFDECSADIR)/*.c $(FFDECSADIR)/*.h
-	@$(MAKE) COMPILER="$(CXX)" FLAGS="$(CSAFLAGS) -march=$(CPUOPT)" PARALLEL_MODE=$(PARALLEL) -C $(FFDECSADIR) all
+	@$(MAKE) COMPILER="$(CXX)" FLAGS="$(CXXFLAGS)" PARALLEL_MODE=$(PARALLEL) -C $(FFDECSADIR) all
 
 $(I18Npot): $(shell grep -rl '\(tr\|trNOOP\)(\".*\")' *.c $(SYSDIR))
 	xgettext -C -cTRANSLATORS --no-wrap --no-location -k -ktr -ktrNOOP --msgid-bugs-address='<noone@nowhere.org>' -o $@ $^
