Index: feeds/packages/libs/boost/Makefile
===================================================================
--- feeds/packages/libs/boost/Makefile	(revision 26761)
+++ feeds/packages/libs/boost/Makefile	(working copy)
@@ -143,6 +143,24 @@
 define Build/Configure
 endef
 
+ifeq ($(CONFIG_PACKAGE_zlib),y)
+CONFIGURE_ARGS+= \
+	-sZLIB_INCLUDE=$(STAGING_DIR)/usr/include \
+	-sZLIB_LIBPATH=$(STAGING_DIR)/usr/lib
+else
+CONFIGURE_ARGS+= \
+	-sNO_ZLIB=1
+endif 
+
+ifeq ($(CONFIG_PACKAGE_bzip2),y)
+CONFIGURE_ARGS+= \
+	-sBZIP2_INCLUDE=$(STAGING_DIR)/usr/include \
+	-sBZIP2_LIBPATH=$(STAGING_DIR)/usr/lib
+else
+CONFIGURE_ARGS+= \
+	-sNO_BZIP2=1
+endif
+
 CONFIGURE_PREFIX:=$(PKG_INSTALL_DIR)
 define Build/Compile
 	( cd $(PKG_BUILD_DIR) ; \
@@ -173,9 +191,6 @@
 			$(if $(CONFIG_PACKAGE_boost-test),--with-test) \
 			$(if $(CONFIG_PACKAGE_boost-thread),--with-thread) \
 			$(if $(CONFIG_PACKAGE_boost-wave),--with-wave) \
-			\
-			$(if $(CONFIG_PACKAGE_boost-iostreams),-sNO_BZIP2=1 -sZLIB_INCLUDE=$(STAGING_DIR)/usr/include \
-				-sZLIB_LIBPATH=$(STAGING_DIR)/usr/lib) \
 			install \
 	)
 endef
