Index: include/cmake.mk
===================================================================
--- include/cmake.mk	(revision 26822)
+++ include/cmake.mk	(working copy)
@@ -21,8 +21,10 @@
   CMAKE_CXX_COMPILER_ARG1:=$(filter-out ccache,$(TARGET_CXX))
 endif
 
+CONFIGURE_PATH = .
+
 define Build/Configure/Default
-	(cd $(PKG_BUILD_DIR); \
+	(cd $(PKG_BUILD_DIR)/$(CONFIGURE_PATH); \
 		CFLAGS="$(TARGET_CFLAGS) $(EXTRA_CFLAGS)" \
 		CXXFLAGS="$(TARGET_CFLAGS) $(EXTRA_CFLAGS)" \
 		cmake \
@@ -46,7 +48,6 @@
 			-DCMAKE_STRIP=: \
 			-DCMAKE_INSTALL_PREFIX=/usr \
 			$(CMAKE_OPTIONS) \
-		. \
 	)
 endef
 
